articles/application-insights/application-insights-correlation.md

```diff
@@ -73,7 +73,7 @@ Application Insights defines the [extension](https://github.com/lmolkova/correla
 
 ### W3C Distributed Tracing
 
-We are transitioning to (W3C Distributed tracing format)[https://w3c.github.io/distributed-tracing/report-trace-context.html]. It defines:
+We are transitioning to [W3C Distributed tracing format](https://w3c.github.io/distributed-tracing/report-trace-context.html). It defines:
```

Comment:
@SergeyKanzhelev This link is broken as well, where should it go? To https://w3c.github.io/trace-context/?